web_favicon/security/web_favicon_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="group_edit_favicon" model="res.groups">
```

Comment:
I agree with @dreispt, remove the group please and use the one provided by Odoo.